commands/logs_test.go

```diff
@@ -0,0 +1,26 @@
+package commands
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+
+	"github.com/docker/docker/pkg/stdcopy"
+	"github.com/mesg-foundation/core/container"
+)
+
+func TestLogsCmdRunE(t *testing.T) {
+	m := newMockExecutor()
+	c := newLogsCmd(m)
+
+	// create reader for docker stdcopy
+	buf := new(bytes.Buffer)
+	w := stdcopy.NewStdWriter(buf, stdcopy.Stdout)
+	w.Write([]byte("core: 2018-01-01 log\n"))
+
+	m.On("Status").Return(container.RUNNING, nil)
+	m.On("Logs").Return(ioutil.NopCloser(buf), nil)
```

Comment:
we still need to test if the logs are correctly piped to output streams, otherwise having this test will have no meaning

Reply:
This test doesn't check the output. I've just added one test ( as an example ) for checking StartCmdTest output. The rest will be in another PR.

Reply:
yes, but in this case what is this test for exactly? i cannot see

Reply:
It tests if after Stats returns `running` Logs was called.

I need to add more of this in the feature.

For example if status return `STOPPED` the logs shouldnt be called, etc...